frontend/src/metabase/dashboard/components/Dashboard.jsx

```diff
@@ -65,19 +65,20 @@ export default class Dashboard extends Component {
 
     async componentDidMount() {
         this.loadParams();
+        const { addCardOnLoad, selectedDashboard, fetchDashboard, fetchCards, addCardToDashboard, onChangeLocation } = this.props;
 
         try {
-            await this.props.fetchDashboard(this.props.selectedDashboard);
-            if (this.props.addCardOnLoad) {
+            await fetchDashboard(this.props.selectedDashboard);
+            if (addCardOnLoad != null) {
```

Comment:
This is equivalent to `addCardOnLoad !== null && addCardOnLoad !== undefined`. It's the one place it's acceptable to use `!=` in JavaScript.